docs/diginstroom/sip/2.1/profiles/film.md

```diff
@@ -10,13 +10,14 @@ Editor's Draft
 {: .label .label-yellow }
 # Profile: Film 
 
-The film profile supports the ingest of digitised film consisting of one or more image and/or audio reels. 
+The film profile supports the ingest of digitised film stored on one or more image and/or audio reels. 
 This profile dictates how the media files (in file formats such as ZIP, MOV and DPX), their metadata, and the relationships between them, should be expressed and organized.
 
-It mainly applies the [DCTERMS metadata schema](https://www.dublincore.org/schemas/xmls/qdc/dcterms.xsd) for descriptive metadata and allows extensions using [Schema.org](https://schema.org), thereby resembling the [Material Artwork profile](https://data.hetarchief.be/id/sip/2.1/material-artwork). 
+It mainly applies the [DCTERMS metadata schema](https://www.dublincore.org/schemas/xmls/qdc/dcterms.xsd) for descriptive metadata and allows extensions using [Schema.org](https://schema.org), thereby resembling the [Basic profile](https://data.hetarchief.be/id/sip/2.1/basic). 
 
 Its additions lie in the introduction of a separate representation to denote the physical carrier(s) (a so-called 'carrier representation') and custom film-specific metadata (using `<premis:significantProperties>` elements in the package PREMIS file) to describe physical aspects of this/these carrier(s).
-This carrier representation was added to facilitate the description of the physical carrier(s), since the PREMIS metadata schema itself doesn't offer this possibility. 
+This carrier representation was added to facilitate the description of the physical carrier(s), since the PREMIS metadata schema itself doesn't offer this possibility directly.
+Please note, however, that the carrier representation as such is not reflected by a representation folder in the `representations` directory, given that it is used purely for the addition of descriptive metadata about the carrier(s).
```

Comment:
Now I'm following that we are talking about a `premis:representation`.

"Its additions lie in the introduction of a (a so-called 'carrier representation' in PREMIS to denote the physical carrier(s) )"

of

"Its additions lie in the introduction of a separate PREMIS representation to denote the physical carrier(s) (a so-called 'carrier representation')"

?